consoles/sshVirtsh.pm

```diff
@@ -112,6 +112,22 @@ sub _init_xml ($self, $args = {}) {
     $elem = $doc->createElement('pae');
     $features->appendChild($elem);
 
+    ## Following features are required for s390's SIE instruction
+    # https://share.confex.com/share/117/webprogram/Handout/Session9564/9564%20-%20zVM%20Security%20and%20Integrity.pdf p6
+    # https://qemu.readthedocs.io/en/latest/system/s390x/vfio-ap.html#start-interpretive-execution-sie-instruction
+    
+    if ( defined $bmwqemu::vars{S390_SIE_AP_MATRIX_PATH} ){
```

Comment:
```suggestion
    # enable features required for s390's SIE instruction, see
    # - https://share.confex.com/share/117/webprogram/Handout/Session9564/9564%20-%20zVM%20Security%20and%20Integrity.pdf#page=6
    # - https://qemu.readthedocs.io/en/latest/system/s390x/vfio-ap.html#start-interpretive-execution-sie-instruction
    if ( defined $bmwqemu::vars{S390_SIE_AP_MATRIX_PATH} ){
```
(Note the referencing of the pdf page via `#page=N`.)

Or just make it a function called `enable_features_requires_for_s390_sie_instruction` (of course keeping the comments with references).